torch/_inductor/fx_passes/mkldnn_fusion.py

```diff
@@ -38,6 +38,81 @@
     _linear_args = [Arg() for _ in range(6)]
     _conv_transpose_args = [Arg() for _ in range(11)]
 
+    def _is_valid_group_gemm_fusion(computation_nodes):
```

Comment:
Add an illustration of such a pattern as code doc?

Reply:
Added the code doc for this check function.